src/app/root.js

```diff
@@ -17,6 +17,9 @@ function rootReducer(state = {}, action) {
       case actionsList.logOut:
         draftState.bakery = null;
         break;
+      case actionsList.collapsibleSidebar:
+        draftState.collapsibleSidebar = action.payload;
```

Comment:
At first I thought we should put UI state in it's own top level key but the `root` state doesn't change often outside of these sidebar changes so we can look at that as a performance optimization later on if necessary.